lib/osut/utils.rb

```diff
@@ -652,43 +648,35 @@ def genShade(model = nil, subs = OpenStudio::Model::SubSurfaceVector.new)
   ##
   # Generates an internal mass definition and instances for target spaces.
   #
-  # @param model [OpenStudio::Model::Model] a model
-  # @param sps [Array<OpenStudio::Model::Space>] target spaces
+  # @param sps [OpenStudio::Model::SpaceVector] target spaces
   # @param ratio [Double] internal mass surface / floor areas
   #
   # @return [Bool] true if successful
   # @return [Bool] false if invalid input (see logs)
-  def genMass(model = nil, sps = [], ratio = 2.0)
-    # This is largely adapted from OpenStudio-Standards
+  def genMass(sps = OpenStudio::Model::SpaceVector.new, ratio = 2.0)
```

Comment:
Switch from input _Arrays_ to _ObjectVectors_, whenever possible. Lightens argument validation.